### PR TITLE
Feature/reviews 

### PR DESCRIPTION
**Pull request by illja-popovics**
Added "reviews" section into the main project.
The cards are responsive on mobile screens and adaptive on tablet/ desktop.
I have also added a scroll overflow, if the text will be bigger in the future.

I had to reset the top padding of the section, so that the distance between sections is correct.
The pictures are retina ready. 
Please let me know if you find any issues.

### CODE REVIEW

File: src/partials/reviews.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            <p class="review-card-profile-subtitle">Student</p>
+          </div>
+        </div>
+        <p class="review-card-text">
+          My experience at the language school has been nothing short of
+          amazing. The teachers are dedicated and the learning environment is
+          stimulating. I feel more confident in my English skills.
+        </p>
+      </li>
+      <li class="review-card card-2">
```
Comment (olexandrd) [inline src/partials/reviews.html:38]: please remove card-_N_ classes from  review-cards

File: src/css/reviews.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+@media only screen and (min-width: 1280px) {
+   .review-card {
+        width: calc((100% - 2*32px) / 3);
+  }
+}
+
+@media only screen and (max-width: 768px) {
+    .card-2{
+        display:none
+    }
```
Comment (olexandrd) [inline src/css/reviews.css:95]:
```
    review-cards-wrapper review-card:not(:first-child) {
        display: none;
    }
```

File: src/css/reviews.css [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+@media only screen and (max-width: 1279px) {
+    .card-3{
+        display: none;
+    }
```
Comment (olexandrd) [inline src/css/reviews.css:101]:
```
    review-cards-wrapper review-card:not(:first-child):not(:nth-child(2)) {
        display: none;
    }
```

**Review by olexandrd — CHANGES_REQUESTED** [review]
please change approach for hiding review cards, left examples on comments

File: src/css/reviews.css (hunk 2 above)
Comment (olexandrd) [reply src/css/reviews.css:95]: for hiding all except first card

File: src/css/reviews.css (hunk 3 above)
Comment (olexandrd) [reply src/css/reviews.css:101]: for hiding all except first and second card

Comment (illja-popovics) [reply src/css/reviews.css]: Hi! I have changed the code, could you please take a look?

**Comment by illja-popovics** [comment]
Hi! I have changed the code according to the comments. Could you please take a look?

File: src/partials/reviews.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        <div class="review-card-profile">
+          <img
+            class="review-card-profile-picture"
+            srcset="
+              /src/images/students/anna.webp    1x,
+              /src/images/students/anna@2x.webp 2x,
+              /src/images/students/anna@4x.webp 4x
+            "
+            src="/src/images/students/anna.webp"
+            alt="picture of a student"
```
Comment (kazamov) [inline src/partials/reviews.html:23]: Personalize the alternative text, `Picture of a student Anna` etc.

**Review by kazamov — CHANGES_REQUESTED** [review]

**Comment by illja-popovics** [comment]
Hi! The branch is now up-to-date, and the suggestions about basic styles were implemented and tested. Please recheck and merge when possible. Thank you!

**Review by Sysqwerty — APPROVED** [review]

**Review by olexandrd — APPROVED** [review]

**Pull request merged**